### PR TITLE
[ZH] Update `osu!team`, `BN`, `NAT`, and `GMT`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] #10092 Merged

### CODE REVIEW

**Review by CloneWith — APPROVED** [review]

File: wiki/People/Global_Moderation_Team/zh.md [hunk 1]
```diff
@@ -53,6 +51,21 @@ tags:
 3. 不要在自己不知道答案是否正确的时候回答问题，特别是在论坛上。提出建议是好的，但误导人可没有任何好处。如果你不知道如何解决问题，请让其他人回答并向他们学习。
 4. 不要做得太过火。团队需要你持之以恒地贡献，而不需要超负荷地支出。众所周知，这会让你精疲力尽，变得失去耐心和违反承诺。放轻松！
 
+## 特权
+
+GM 们可以获得以下特权：
```
Comment (RushFTK) [inline wiki/People/Global_Moderation_Team/zh.md:56]:
```suggestion
全局管理员们可以获得以下特权：
```

你在前面的译文中并没有诸如"全局管理员(GM)"的翻译。用的大多是"全局管理团队的所有成员"的说法

File: wiki/People/Global_Moderation_Team/zh.md [hunk 2]
```diff
@@ -53,6 +51,21 @@ tags:
...
+
+- **[osu! 支持者](/wiki/osu!supporter)**：获得一个时长等于任期的 osu! 支持者标签。
+- **职位奖牌**：从第 3 年开始，可获得一个显示任期年份的[主页奖牌](/wiki/Community/Profile_badge)。
+  ![3 年 GMT 奖牌](https://assets.ppy.sh/profile-badges/GMT3y.png "3 年 GMT 奖牌") ![4 年 GMT 奖牌](https://assets.ppy.sh/profile-badges/GMT4y.png "4 年 GMT 奖牌") ![5 年 GMT 奖牌](https://assets.ppy.sh/profile-badges/GMT5y.png "5 年 GMT 奖牌")
+
+## 离开全局管理团队 (GMT)
+
+如果 GM 选择离开团队，他们将根据以下情况获得特权：
```
Comment (RushFTK) [inline wiki/People/Global_Moderation_Team/zh.md:64]:
```suggestion
如果全局管理员选择卸任，他们将根据以下情况获得特权：
```
 chooses to depart from the team 更多是一种正常方式的离职，我认为使用卸任更好。

File: wiki/People/Global_Moderation_Team/zh.md [hunk 3]
```diff
@@ -53,6 +51,21 @@ tags:
...
+
+- **[osu! 名人堂](/wiki/People/osu!_Alumni) 玩家头衔**：授予给在团队中至少任职 1 年，或是在短时间内为团队做出重大贡献的成员。
+- **osu! 支持者**：如果团队成员获得了名人堂头衔，他们也将获得 2 倍任职时长的 osu! 支持者标签。
```
Comment (RushFTK) [inline wiki/People/Global_Moderation_Team/zh.md:67]:
```suggestion
- **osu! 支持者**：与获得名人堂头衔的条件相同，他们将获得 2 倍任职于全局管理团队时长的 osu! 支持者标签。
```

一个用户可以符合资格但是拒绝认领名人堂头衔，所以原文的只是条件相同，我认为这样翻译更好。

如果翻译成获得之后同时获得，有种两种奖励并非相互独立的意思。

**Review by RushFTK — APPROVED** [review]
Majorly fines for me, just some option that can be imporved are given on review.

**Review by TicClick — APPROVED** [review]

**Pull request merged**